src/libs/OptionsListUtils.ts

```diff
@@ -1525,22 +1464,12 @@ function getFilteredOptions(
 
 /**
  * Build the options for the Share Destination for a Task
- * *
- * @param {Object} reports
- * @param {Object} personalDetails
- * @param {Array<String>} [betas]
- * @param {String} [searchValue]
- * @param {Array} [selectedOptions]
- * @param {Array} [excludeLogins]
- * @param {Boolean} [includeOwnedWorkspaceChats]
- * @returns {Object}
- *
  */
 
 function getShareDestinationOptions(
-    reports,
-    personalDetails,
-    betas = [],
+    reports: Record<string, Report>,
+    personalDetails: OnyxCollection<PersonalDetails>,
```

Comment:
```suggestion
    reports: OnyxCollection<Report>,
    personalDetails: OnyxEntry<PersonalDetailsList>,
```

`reports` should be an Onyx collection.

PersonalDetails is not a Onyx Collection.

Reply:
Also add return type!